### DISCUSSION TITLE
💋 【共学营打卡】【第1次 Milestone】【Move-0x01】Run the dev starcoin node & get faucet

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#105] [Projects-Others]
教程见：
https://github.com/NonceGeek/Web3-dApp-Camp/tree/main/move-dapp#12-setup-a-node

回复截图打卡

**Comment by leeduckgo** [comment]
<img width="1438" alt="image" src="https://user-images.githubusercontent.com/12784118/180641515-cca6c8aa-9ff6-4c07-888a-6d9c4377010f.png">

**Comment by pochenai** [comment]
![1](https://user-images.githubusercontent.com/13568446/180716517-c3ccc547-a45f-4e31-bc7b-4b71ebca3485.png)

**Reply by leeduckgo** [reply]
First Kill: ¥88.88

**Comment by qiwihui** [comment]
![Screen Shot 2022-07-27 at 4 48 39 PM](https://user-images.githubusercontent.com/3297411/181204674-2f1b45c9-4940-49be-8f06-c6c6a8f81920.png)
![Screen Shot 2022-07-27 at 4 48 09 PM](https://user-images.githubusercontent.com/3297411/181204659-e3482a56-1eaa-4cb3-bcd3-f3ca0bda09c5.png)

**Comment by hqwangningbo** [comment]
<img width="1157" alt="image" src="https://user-images.githubusercontent.com/57781136/181238961-a0782392-1693-4123-9ddb-5793c08c93d6.png">


<img width="1151" alt="image" src="https://user-images.githubusercontent.com/57781136/181238864-9f4c9e05-bca9-4346-8c05-f4cd1f703c93.png">

<img width="1142" alt="image" src="https://user-images.githubusercontent.com/57781136/181239165-b23123bf-85af-4a07-808f-25492728731a.png">

**Reply by 0xKT** [reply]
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/74288668/181415479-ce49ae07-e595-4f88-9a2d-5cbcb881f360.png">

**Comment by leeduckgo** [comment]
GoodGood

**Comment by 99Kies** [comment]
![image](https://user-images.githubusercontent.com/45918704/181439664-88da3016-3d29-4361-a9f8-2d905bd8f61c.png)
![image](https://user-images.githubusercontent.com/45918704/181440373-0803788e-3efb-4d6d-8bb9-98aad98d0e62.png)
![image](https://user-images.githubusercontent.com/45918704/181440504-e8c37ef2-ac31-42b4-ba4a-9a3c4ba94f52.png)

**Comment by gofenix** [comment]
<img width="1920" alt="image" src="https://user-images.githubusercontent.com/6822558/181447341-a772bcb9-4af5-40dc-9d48-a55b5337a716.png">


https://www.yuque.com/docs/share/d9b03547-6711-4776-89af-c365921fa574?# 《starcoin challenges》

**Reply by armatrix** [reply]
<img width="1405" alt="image" src="https://user-images.githubusercontent.com/62870322/187229935-9d7985e6-7b58-4a0d-805b-f9b8818bf3bc.png">

**Comment by ZacharyWulven** [comment]
<img width="987" alt="截屏2022-10-10 00 44 19" src="https://user-images.githubusercontent.com/10501049/194769034-53a96179-0855-4e21-adf5-b9fc7efb82c4.png">

**Comment by sharkqwy** [comment]
<img width="722" alt="截屏2022-10-17 15 18 34" src="https://user-images.githubusercontent.com/1372795/196113122-7b18d821-b708-42c6-a4e7-6269feccfe77.png">

**Comment by zhaoge5859** [comment]
![WechatIMG11](https://user-images.githubusercontent.com/116735858/198822466-3e2f026e-a48e-4832-8ec6-654234438bc8.jpeg)
![WechatIMG12](https://user-images.githubusercontent.com/116735858/198822473-a9723306-bced-4676-a244-2ac2ccf99ab8.jpeg)
![WechatIMG13](https://user-images.githubusercontent.com/116735858/198822477-ab7483fd-047d-47ff-b0d0-f734da39ce1a.jpeg)

**Reply by zhaoge5859** [reply]
这算成功了吗？

**Comment by livelybug** [comment]
![image](https://user-images.githubusercontent.com/13627060/200171202-6456a804-7caf-417e-8790-dbd51ea47539.png)

**Comment by pingp76** [comment]
![WX20221106-204742@2x](https://user-images.githubusercontent.com/8874696/200171616-17bc3511-de4e-477a-8cbc-87ec6cbab522.png)

**Comment by MagicGordon** [comment]
<img width="1676" alt="7061" src="https://user-images.githubusercontent.com/19465870/200175272-51b0d3be-9157-442e-b5a3-5aad47933d55.png">

**Comment by Hosuke** [comment]
> 教程见： https://github.com/WeLightProject/Web3-dApp-Camp/blob/main/move-dapp/CryptoXiuXian%20in%20Move%20Lang.md#12-setup-a-node
> 
> 回复截图打卡


更新有效链接🔗
https://github.com/NonceGeek/Web3-dApp-Camp/tree/main/move-dapp

**Comment by Hosuke** [comment]
<img width="815" alt="image" src="https://user-images.githubusercontent.com/6568873/200256117-b294459d-a3d8-4af1-bbb6-935f4d7c8ee2.png">
<img width="671" alt="image" src="https://user-images.githubusercontent.com/6568873/200256731-c26db52c-5876-4f0e-b9bb-5bf9aaf0c2aa.png">

**Comment by v1xingyue** [comment]
![image](https://user-images.githubusercontent.com/974169/200277672-ec50d970-9793-4a70-8fb8-269029869e26.png)

Done.

**Comment by xiegudong45** [comment]
![image](https://user-images.githubusercontent.com/30063330/200344667-6f557d40-4be2-457c-a522-441ea5f9b8b4.png)
![image](https://user-images.githubusercontent.com/30063330/200344992-92d484de-fdf0-45c1-a1f9-d5a32e760da2.png)

**Comment by Demian101** [comment]
<img width="812" alt="image" src="https://user-images.githubusercontent.com/33189338/200459744-65295351-3fde-418d-8e94-1170cee32af2.png">
<img width="834" alt="image" src="https://user-images.githubusercontent.com/33189338/200459784-dca31ceb-f64e-474a-b472-a50f83457f38.png">

**Comment by valerywai** [comment]
<img width="759" alt="image" src="https://user-images.githubusercontent.com/53373706/200591919-092fa02a-09d7-4acc-a009-5bee801c3c19.png">
<img width="759" alt="image" src="https://user-images.githubusercontent.com/53373706/200592990-cc1d4bb4-31b3-48bc-bb01-c83fa9ca9a94.png">

**Comment by geometryolife** [comment]
![image](https://user-images.githubusercontent.com/54882546/201508286-13ec6c38-7901-4ffa-94a8-45ef8242ebd1.png)
![image](https://user-images.githubusercontent.com/54882546/201508292-4ae9e806-d08a-4c20-9b14-97c639319e4f.png)
![image](https://user-images.githubusercontent.com/54882546/201508300-7409f43f-23ea-45ab-ba8f-1ba756c578b5.png)

**Comment by X0JW** [comment]
![26db56160c08e7581e47b4348bd5a66](https://user-images.githubusercontent.com/116423559/203099555-be36d4a0-d66f-4d9a-a5aa-69bc922b079b.jpg)
![star coin 安装](https://user-images.githubusercontent.com/116423559/203100733-c8d0ef51-2afb-409f-894e-e5906ad7e2cc.jpg)

课程结束了才学会安装dev网络

**Comment by hi-chenerge** [comment]
![starcoin节点启动](https://user-images.githubusercontent.com/82131587/211196297-b33e8a43-77b8-4c9b-8a48-808eaf72ff0e.jpg)
![获取token](https://user-images.githubusercontent.com/82131587/211196301-4eaac779-192e-4b61-a8a4-3d0ef78500fe.jpg)
![shwoaccount](https://user-images.githubusercontent.com/82131587/211196305-40f1d617-96df-4258-b227-36d62265e7e7.jpg)

**Reply by hi-chenerge** [reply]
pic1: 启动成功
pic2:获取代币
pic3:查看account

**Comment by 0xsu** [comment]
<img width="507" alt="image" src="https://user-images.githubusercontent.com/101964290/212122641-9f94c2b0-065d-4f3e-b8b5-df55610c0a18.png">

**Comment by Whitehare2023** [comment]
![64b173062e18940af02a47920006f23](https://user-images.githubusercontent.com/124434634/216921812-c3f7a967-304a-43c4-b1f1-b28f2c2ccfaa.png)
![d1c63e75bd82580188c629819eb54b8](https://user-images.githubusercontent.com/124434634/216921835-4ccefe50-3948-4e82-9c3d-b78a4141e6cd.png)

**Comment by baidang201** [comment]
<img width="832" alt="image" src="https://user-images.githubusercontent.com/3405789/218300843-901db0fa-3c22-4da1-896e-7bedf09e259c.png">
<img width="1123" alt="image" src="https://user-images.githubusercontent.com/3405789/218300868-d2a28e5e-97c0-4aff-b2eb-50132fb1b5c1.png">
<img width="627" alt="image" src="https://user-images.githubusercontent.com/3405789/218300878-f3400abd-30dd-4190-a644-d8a49719befb.png">

**Comment by DivenYoung** [comment]
![1676200691525](https://user-images.githubusercontent.com/22409395/218307844-544b6590-1f59-4eb5-8b36-aa668dbd2fe1.jpg)
<img width="1131" alt="CleanShot 2023-02-12 at 19 17 11@2x" src="https://user-images.githubusercontent.com/22409395/218307852-6a498441-83ed-4626-8441-4e9af8958870.png">
